site/content/docs/5.0/getting-started/accessibility.md

```diff
@@ -47,7 +47,7 @@ For visually hidden interactive controls, such as traditional "skip" links, use
 
 Bootstrap includes support for the [`prefers-reduced-motion` media feature](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion). In browsers/environments that allow the user to specify their preference for reduced motion, most CSS transition effects in Bootstrap (for instance, when a modal dialog is opened or closed, or the sliding animation in carousels) will be disabled.
 
-Alternatively, the `scroll-behavior` property enables smooth scroll if the user did not ask for reduced motion.
+Bootstrap also enables smooth scroll if the user did not ask for reduced motion, using the `scroll-behavior` property.
```

Comment:
Perhaps a bit verbose, but I'd go for something along the lines of:

```
On browsers that support the `prefers-reduced-motion`, and where the user has *not* explicitly signaled that they'd prefer reduced motion (i.e. where `prefers-reduced-motion: no-preference`), Bootstrap enables smooth scrolling using the `scroll-behavior` property. 
```

Reply:
Very clear, thanks!